### PR TITLE
Build CI/CD for test and push image Production

### PR DESCRIPTION
**Pull request by nnpvaan**
### What does this do?
Build CI/CD for test and push image Production

### Which issue(s) does this PR fix/relate to?
https://github.com/Ion-Mobility/consumer-project/issues/240

### CODE REVIEW

File: .github/workflows/workflow-prod.yml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        with:
+          go-version: 1.19.4
+
+      - name: Run all tests
+        run: |
+          go test -mod=vendor -v -race -count 1 -tags test $(go list ./... | grep -v 'vendor\|cmd')
+
+  build-images:
+    name: "Build and Push image to GCR"
+    runs-on: ubuntu-latest
```
Comment (wilianto) [inline .github/workflows/workflow-prod.yml:46]: should it depends on test? only if test pass then we build the image.

File: .github/workflows/workflow-prod.yml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            BUILD_CONTEXT=docker/vernemq
+          fi
+          docker build \
+            --no-cache \
+            $BUILD_ARGS \
+            -t ${{ secrets.IMAGE_PROJECT }}/${{ secrets.MF_DOCKER_IMAGE_NAME_PREFIX }}/${{ matrix.service }} \
+            -f $DOCKERFILE_PATH $BUILD_CONTEXT
+      
+      - name: Push image
+        run: docker push ${{ secrets.IMAGE_PROJECT }}/${{ secrets.MF_DOCKER_IMAGE_NAME_PREFIX }}/${{ matrix.service }}
```
Comment (wilianto) [inline .github/workflows/workflow-prod.yml:98]: should we put tag here?

File: .github/workflows/workflow-prod.yml (hunk 1 above)
Comment (nnpvaan) [reply .github/workflows/workflow-prod.yml:46]: @wilianto I fixed that, please help me check again

File: .github/workflows/workflow-prod.yml (hunk 2 above)
Comment (wilianto) [reply .github/workflows/workflow-prod.yml:99]: oh ok, if default is latest. then should ok.

**Review by wilianto — APPROVED** [review]
lgtm!

**Review by nintran52 — APPROVED** [review]
lgtm!

**Pull request merged**